tests/test_dummy_converter.py

```diff
@@ -1,10 +1,9 @@
 import pytest
-from converter.converter import BeamConfig, MatConfig, DetectConfig, GeoConfig
+import tempfile
+from os import path
+from converter.converter import BeamConfig, MatConfig, DetectConfig, GeoConfig, DummmyParser, Runner
 
-
-def test_default_beam_config_str() -> None:
-    """Test if the default BeamConfig str representation is correct."""
-    assert str(BeamConfig()) == """
+BEAM_STR = """
```

Comment:
Avoid uppercase, follow general naming conventions in python, see nice
https://www.python.org/dev/peps/pep-0008/#global-variable-names

Reply:
Changed them the uppercase to capitalization and added an an underscore before name (_Beam_str). Not sure if that's what we wanted.